numba/parfor.py

```diff
@@ -2166,7 +2167,7 @@ def _setitem_to_parfor(self, equiv_set, loc, target, index, value, shape=None):
         index_var, index_var_typ = self._make_index_var(
                  scope, index_vars, body_block)
         parfor = Parfor(loopnests, init_block, {}, loc, index_var, equiv_set,
-                        ('setitem',), self.flags)
+                        ('setitem', ''), self.flags)
```

Comment:
What is the `''` for?

Reply:
It's a placeholder for additional information. In some parallel patterns this could be a dict of options, others it might be a string with information about the scopes of pattern, and in other's it's a user readable string. I've not decided what to put in there yet for `setitem` Parfors as I've not got all the tracking for them worked out in the diagnostics tool.